src/scripts/main.js

```diff
@@ -1,5 +1,24 @@
 'use strict';
 
 document.addEventListener('click', (e) => {
-  // write code here
+  const wall = document.querySelector('.wall');
+  const spider = document.querySelector('.spider');
```

Comment:
It's not efficient to query the DOM every time a click event is triggered. You can move these queries outside of the event handler, as the wall and spider elements are not changing.